### DISCUSSION TITLE
Able to help me get StarStruck Achievement?

### DISCUSSION THREAD

**Discussion by Coolbreeze151** [community/community#140023] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to achieve 16 stars for my repo to achieve starstruck. Any help would be great. Any tips or advice would be great too! Thanks!

https://github.com/Coolbreeze151/BackendRoadmap

**Comment by ferid333** [comment]
Hello @Coolbreeze151 
How did you get this : Arctic Code Vault Contributor ?

**Reply by Coolbreeze151** [reply]
Hi @ferid333 it was a one off badge in a previous year issued to users who may have had their repos archived. If I understand it was discontinued.

**Comment by Coolbreeze151** [comment, marked as answer]
Adding a suggested answer here to see if as a owner I can indicate it as answered.

**Reply by Coolbreeze151** [reply]
Closing this issue. But anyone who would want to exchange stars with repos. Do go ahead and reply in this chat and we can exchange stars!

**Comment by ferid333** [comment]
I can exchange stars

**Reply by Coolbreeze151** [reply]
https://github.com/Coolbreeze151/BackendRoadmap

**Reply by ferid333** [reply]
https://github.com/ferid333/InstaNinja

**Comment by ferid333** [comment]
@Coolbreeze151 Hello again. How can I get Pair Extraordinaire achievement ?

**Comment by diobenu2silva** [comment]
I can also exchange stars. Please help each other.

https://github.com/diobenu2silva/pumpfun_SC
https://github.com/diobenu2silva/zeroGpt4
https://github.com/diobenu2silva/solArmy_staking_SC
https://github.com/diobenu2silva/triangular_arbitrage_multi_exchange
